pkgs/tools/misc/hhpc/default.nix

```diff
@@ -0,0 +1,26 @@
+{stdenv, fetchFromGitHub, xorg, pkgconfig}:
+
+stdenv.mkDerivation rec {
+  name = "hhpc-${version}";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "aktau";
+    repo = "hhpc";
+    rev = "v${version}";
+    sha256 = "1djsw1r38mh6zx0rbyn2cfa931hyddib4fl3i27c4z7xinl709ss";
+  };
+
+  buildInputs = [ xorg.libX11 pkgconfig ];
```

Comment:
Please add pkgconfig to nativeBuildInputs instead of buildInputs. See eg. https://nixos.org/wiki/CrossCompiling